apollo-runtime/src/main/java/com/apollographql/apollo/internal/subscription/RealSubscriptionManager.kt

```diff
@@ -25,12 +25,12 @@ import java.util.concurrent.CopyOnWriteArrayList
 import java.util.concurrent.Executor
 import java.util.concurrent.TimeUnit
 
-class RealSubscriptionManager(customScalarAdapters: CustomScalarAdapters,
+class RealSubscriptionManager(private val customScalarAdapters: CustomScalarAdapters,
                               transportFactory: SubscriptionTransport.Factory,
-                              connectionParams: SubscriptionConnectionParamsProvider,
-                              dispatcher: Executor, connectionHeartbeatTimeoutMs: Long,
-                              responseNormalizer: Function0<ResponseNormalizer<Map<String, Any>>>,
-                              autoPersistSubscription: Boolean
+                              private val connectionParams: SubscriptionConnectionParamsProvider,
+                              private val dispatcher: Executor, private val connectionHeartbeatTimeoutMs: Long,
+                              private val responseNormalizer: Function0<ResponseNormalizer<Map<String, Any>>>,
```

Comment:
should it be replaced with Kotlin functional type?

Reply:
Oh yes ! We have a bunch of them everywhere. I'll replace this one.